dashboard/lib/logic/qualified_task.dart

```diff
@@ -34,6 +34,9 @@ class QualifiedTask {
   final String? stage;
   final String? task;
 
+  /// Whether this task originated as a `bringup: true` task.
+  bool get isBringUp => pool == 'luci.flutter.staging';
```

Comment:
Even better: make it `final bool isBringup` and then do `isBringup = task.isFlaky` in the constructor. No need to use the pool name as a proxy.